tmt/steps/provision/testcloud.py

```diff
@@ -225,8 +294,8 @@ def show(
 
         super().show(keys=super_keys, verbose=verbose, logger=logger)
 
-        logger.info('memory', f"{self.memory} MB", 'green')
-        logger.info('disk', f"{self.disk} GB", 'green')
+        logger.info('memory', f"{self.memory or DEFAULT_MEMORY}", 'green')
+        logger.info('disk', f"{self.disk or DEFAULT_DISK}", 'green')
```

Comment:
So it now shows:

```
        memory: 2000 megabyte
        disk: 40 gigabyte
```

Is it intentional to not use common abbreviatios like `MB` or `GB`  here?

Reply:
True, but added a conversion to MB/GB in 90481193

Reply:
Actually, I would leave this for another patch. The change alone would be simple:

```diff
diff --git a/tmt/hardware.py b/tmt/hardware.py
index 8b622123..fcd36437 100644
--- a/tmt/hardware.py
+++ b/tmt/hardware.py
@@ -70,6 +70,7 @@ if TYPE_CHECKING:
 
 #: Unit registry, used and shared by all code.
 UNITS = pint.UnitRegistry()
+UNITS.default_format = '~D'
 
 
 # Special type variable, used in `Constraint.from_specification` - we bound this return value to
```

But it might spawn a set of test changes here and there, and that might be unrelated to testcloud changes here.

Another option we have is to enforce this format here and now for testcloud, but then I'd easily forget to drop it when the patch changing the default format lands.

Reply:
Never mind, that's way too much work. Added right away in 02951887.

Reply:
Never mind once more, I had to undo this. The nice and easy-to-use formatting is available with Pint 0.18 and newer, the best we can get from RPMs is 0.17.1. I filed https://github.com/teemtee/tmt/issues/2410, and I'd leave the more verbose version in for now.